tensorflow/python/keras/applications/efficientnet.py

```diff
@@ -16,9 +16,9 @@
 # pylint: disable=missing-docstring
 """EfficientNet models for Keras.
 
-Reference paper:
-  - [EfficientNet: Rethinking Model Scaling for Convolutional Neural Networks]
-    (https://arxiv.org/abs/1905.11946) (ICML 2019)
+  Reference:
```

Comment:
this should have the same indent as "Reference Paper".

Reply:
fixed this one.